pyreisejl/utility/launchers.py

```diff
@@ -129,6 +129,34 @@ def launch_scenario(self):
         raise NotImplementedError
 
 
+class ClpLauncher(Launcher):
```

Comment:
Minor: should it be `CLP` instead to follow `GLPK`?

Reply:
The Julia project is `Clp.jl`, so I followed that convention.